From 7dd88f975774cb379284a3de819232dcce3ca8fd Mon Sep 17 00:00:00 2001
From: Timo Gurr <timo.gurr@gmail.com>
Date: Tue, 30 Jun 2015 11:33:14 +0200
Subject: [PATCH] Set pkg-config binary as variable

Allow the location of the pkg-config binary to be set using the
parameter PKG_CONFIG. Needed to support cross compilation toolchains.
---
 src/efibootmgr/module.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/efibootmgr/module.mk b/src/efibootmgr/module.mk
index 8b03b92..40eb6f6 100644
--- a/src/efibootmgr/module.mk
+++ b/src/efibootmgr/module.mk
@@ -10,7 +10,7 @@ efibootmgr_LIBS    := efi.o unparse_path.o
 efibootmgr_LIBDIR  := src/lib
 efibootmgr_FULLLIB := \
 	$(patsubst %,$(efibootmgr_LIBDIR)/%,$(efibootmgr_LIBS))
-LIBS = $(shell pkg-config --libs efivar efiboot)
+LIBS = $(shell $(PKG_CONFIG) --libs efivar efiboot)
 
 ALLDEPS += $(efibootmgr_FULLTARGET)
 CLEANLIST += $(efibootmgr_FULLTARGET)
-- 
2.4.5

